composer.json

```
@@ -17,6 +17,7 @@
         "doctrine/inflector": "^1.0",
         "psr/cache": "^1.0",
         "psr/container": "^1.0",
+        "symfony/config": "^2.0 || ^3.0 || ^4.0",
```

```suggestion
        "symfony/config": "^3.4 || ^4.0",
```

I don't think we should support old unmaintained versions. It's just needlessly increasing our maintenance surface.